src/store/reviewSpot/index.js

```diff
@@ -338,13 +345,21 @@ const actions = {
         if (!isValid) {
             return;
         }
-        const response = await dispatch('updateSpotRequest');
-        if (response.DisplayMsg) {
-            // Network issues or server errors could cause the API call to fail.
-            commit('set-error-msg', response.DisplayMsg);
-        } else {
-            commit('set-success-msg', 'Your request was saved successfully');
+        commit('set-loading', true);
+        const uploadedImageURLs = await ImageUploadService.methods.uploadImages(state.SO.uploadImages, state.SO.spotId);
+        if (uploadedImageURLs['success'] === false) {
```

Comment:
`(uploadedImageURLs['success'] === false` can be replaced with `!uploadedImageURLs['success']`